src/addons/Responsive/Responsive.d.ts

```diff
@@ -28,12 +28,12 @@ export interface ResponsiveProps {
 }
 
 export interface ResponsiveOnUpdateData extends ResponsiveProps {
-  width: string;
+  width: number;
 }
 
 export interface ResponsiveWidthShorthand {
-  minWidth: number;
-  maxWidth?: number;
+  minWidth?: number | string;
```

Comment:
`minWidth` is actually optional

Reply:
This is a potentially breaking change (everyone who expects minWidth to be there and has coded against it will now potentially run into linter errors stating that minWidth is optional and should change their code to check for this).